### ISSUE TITLE
carousel: axe fixes

### ISSUE THREAD

**Issue by jaketrent** [pluralsight/design-system#1755]
Enable axe test and fix:

- Elements of role none or presentation should be flagged (presentation-role-conflict)
- - Elements should not have tabindex greater than zero (tabindex)
- - Certain ARIA roles must be contained by particular parents (aria-required-parent)
